src/collection/pages/main/main.tsx

```diff
@@ -290,20 +295,22 @@ const Main: React.FC = () => {
         </div>
       )}
       <div className={"CollectionItemList"}>
-        {cartList.map((x, i) => (
-          <CollectionItem item={x} isEdit={edit} key={i} />
-        ))}
+        {edit ? (tempCartList.map((x, i) => (
+          <CollectionItem item={x} isEdit={edit} key={i} /> 
+        ))) : (cartList.map((x, i) => (
+          <CollectionItem item={x} isEdit={edit} key={i} /> 
+        )))}
```

Comment:
I didn't try, but...

```suggestion
        {(edit ? tempCartList : cartList).map((x, i) => (
          <CollectionItem item={x} isEdit={edit} key={i} />
        ))}
```